draft-ietf-quic-qpack.md

```diff
@@ -304,76 +316,77 @@ Count in order to identify which dynamic table entries can be referenced without
 potentially blocking a stream.  The decoder tracks the Known Received Count in
 order to be able to send Insert Count Increment instructions.
 
-A Header Acknowledgement instruction ({{header-acknowledgement}}) implies that
-the decoder has received all dynamic table state necessary to process
-corresponding the header block.  If the Required Insert Count of the
-acknowledged header block is greater than the current Known Received Count,
-Known Received Count is updated to the value of the Required Insert Count.
+An Entry Acknowledgement instruction ({{entry-acknowledgement}}) implies that
+the decoder has received all dynamic table state necessary to decode the field
+section.  If the Required Insert Count of the acknowledged field section is
+greater than the current Known Received Count, Known Received Count is updated
+to the value of the Required Insert Count.
 
 An Insert Count Increment instruction {{insert-count-increment}} increases the
 Known Received Count by its Increment parameter.  See {{new-table-entries}} for
 guidance.
 
 ## Decoder
 
-As in HPACK, the decoder processes header blocks and emits the corresponding
-header lists. It also processes instructions received on the encoder stream that
-modify the dynamic table.  Note that header blocks and encoder stream
-instructions arrive on separate streams.  This is unlike HPACK, where header
-blocks can contain instructions that modify the dynamic table, and there is no
-dedicated stream of HPACK instructions.
+As in HPACK, the decoder processes a series of representations and emits the
+corresponding field sections. It also processes instructions received on the
+encoder stream that modify the dynamic table.  Note that encoded field sections
+and encoder stream instructions arrive on separate streams.  This is unlike
+HPACK, where field sections can contain instructions that modify the dynamic
+table, and there is no dedicated stream of HPACK instructions.
 
-The decoder MUST emit header fields in the order their representations appear in
-the input header block.
+The decoder MUST emit field lines in the order their representations appear in
+the encoded field section.
 
 ### Blocked Decoding
 
-Upon receipt of a header block, the decoder examines the Required Insert Count.
-When the Required Insert Count is less than or equal to the decoder's Insert
-Count, the header block can be processed immediately.  Otherwise, the stream on
-which the header block was received becomes blocked.
+Upon receipt of an encoded field section, the decoder examines the Required
+Insert Count. When the Required Insert Count is less than or equal to the
+decoder's Insert Count, the field section can be processed immediately.
+Otherwise, the stream on which the field section was received becomes blocked.
 
-While blocked, header block data SHOULD remain in the blocked stream's flow
+While blocked, field section data SHOULD remain in the blocked stream's flow
```

Comment:
Consider writing "encoded field section" instead of "field section".  Header block and header list were distinct things, and they got renamed to "encoded field section" and "field section" respectively.  Therefore "encoded field section" is not a kind of "field section", but a different concept.